### PR TITLE
Clean up Popularity Refresh notifications and adjust poke interval

### PR DESCRIPTION
**Pull request by stacimc**
## Fixes

Some fast-follows after turning on the popularity refresh DAGs in production.

## Description

A few quick fast-follows:

- Updates the poke interval for the audio refresh to 1 minute: in production, it took around 5 minutes for the entire audio refresh to complete, so the 30 minute poke interval is much too high
- Updates the `execution_timeout` on the `TriggerDagRunOperator` to 30 days to match the timeout of the batched updates. (In production, the image batched updates are proceeding well, but the image refresh DAG timed out at this step)
- In the `batched_update` DAG, instead of logging the number of rows completed _and the number remaining_, it now logs the number of rows completed and _the percent completed_, which is more useful for large DAGs like Flickr.
- Formats all the `count`s given in the various log messages and slack notifications to use comma separators (e.g. "Updating 123,456,789 records" instead of "Updating 123456789 records")
- Capitalizes the media type in slack notifications
- Updates the Slack notifications during the popularity constant recalculation steps to read "<Media> Data Refresh Notification" when coming from a data refresh, and "<Media> Popularity Refresh Notification" when coming from the popularity refresh.

## Testing Instructions

Run the audio and image popularity refreshes locally. Inspect the `update_batches` task to see it has the correct timeout and poke interval (to test the audio poke interval, you may need to remove your own `DATA_REFRESH_POKE_INTERVAL` env variable if you have one, as this will always override defaults). Check that the slack notifications and logs are formatted properly.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: catalog/dags/data_refresh/reporting.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def report_status(media_type: str, message: str, dag_id: str):
     slack.send_message(
         text=message,
         dag_id=dag_id,
-        username="Data Refresh Notification",
+        username=f"{dag_id.replace('_', ' ').title()} Notification",
```
Comment (AetherUnbound) [inline catalog/dags/data_refresh/reporting.py:14]: Nice change!

File: catalog/dags/popularity/popularity_refresh_types.py [hunk 2]
```diff
@@ -57,5 +67,9 @@ def __post_init__(self):
         media_type="image",
         refresh_metrics_timeout=timedelta(hours=24),
     ),
-    PopularityRefresh(media_type="audio"),
+    PopularityRefresh(
+        media_type="audio",
+        # Poke every minute, instead of every thirty minutes
+        poke_interval=int(os.getenv("DATA_REFRESH_POKE_INTERVAL", 60)),
```
Comment (AetherUnbound) [inline catalog/dags/popularity/popularity_refresh_types.py:73]: Is there a better way to manage this default? 🤔 It feels weird running the exact same steps as we do for building `REFRESH_POKE_INTERVAL`, but with a different default 😅

**Review by AetherUnbound — APPROVED** [review]
Looks good! Just one other thought regarding defaults, nothing blocking.

File: catalog/dags/popularity/popularity_refresh_types.py (hunk 2 above)
Comment (stacimc) [reply catalog/dags/popularity/popularity_refresh_types.py:73]: Agreed! The problem is that if you set a poke interval via the environment variable, I want it to override all other configurations. So when you set a `poke_interval` in the PopularityRefresh configuration, it _should_ override the normal default, but _not_ the env var. 

REFRESH_POKE_INTERVAL is used in several other places including the DataRefresh configurations. We could have separate environment variables for the, but that seems unnecessary, or we could make them inherit from a common configuration which handles the defaults in one place?

**Review by krysal — APPROVED** [review]
LGTM!

File: catalog/dags/popularity/popularity_refresh_types.py (hunk 2 above)
Comment (AetherUnbound) [reply catalog/dags/popularity/popularity_refresh_types.py:73]: That all makes sense, we can keep it as-is!

**Pull request merged**